tests/unit/Command/DependencyResolverCommandTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OAT\DependencyResolver\Command;
+
+use OAT\DependencyResolver\Extension\Entity\Extension;
+use OAT\DependencyResolver\Extension\ExtensionFactory;
+use OAT\DependencyResolver\FileSystem\FileAccessor;
+use OAT\DependencyResolver\Manifest\DependencyResolver;
+use OAT\DependencyResolver\Repository\RepositoryMapAccessor;
+use OAT\DependencyResolver\TestHelpers\ProtectedAccessorTrait;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+
+/**
+ * Unit tests for DependencyResolverCommand class.
+ */
+class DependencyResolverCommandTest extends TestCase
```

Comment:
Commands should be covered by functional tests instead of unit tests (as we discussed in person). Please refactor

Reply:
This command is done. I have to cover the two other ones...